cmd/crc-embedder/cmd/root.go

```diff
@@ -26,6 +26,10 @@ when building the crc executable for release`,
 }
 
 func init() {
+	err := constants.EnsureBaseDirectoriesExist()
+	if err != nil {
+		fmt.Println("CRC base directories are missing: ", err)
+	}
```

Comment:
Should `return` or `os.Exit` maybe?

Reply:
yes as you wish. It is not production code. crc does `logging.Fatal()`.

Reply:
added `return`

Reply:
If you add return here, it won't add the flag and continue the execution ? I think you should either use logging.Fatal or os.Exit but not return.

Reply:
updated, used `os.Exit(1)`